### ISSUE TITLE
ValueError: year is out of range

### ISSUE THREAD

**Issue by mrpintcom** [picklepete/pyicloud#180]
I'm trying to use pyicloud in the [icloud-photos-downloader](https://github.com/ndbroadbent/icloud_photos_downloader) package to pull all my photos and videos.

However, part way through I encounter the following error:

```/home/luke/Pictures/2016/09/24/IMG_3101-original.JPG already exists.:  55%|▌| 18392/33335 [09:31<07:27, 33.43it/s]Traceback (most recent call last):
  File "./download_photos.py", line 304, in <module>
    download()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "./download_photos.py", line 152, in download
    created_date = photo.created.astimezone(get_localzone())
  File "/usr/local/lib/python2.7/dist-packages/pyicloud/services/photos.py", line 451, in created
    return self.asset_date
  File "/usr/local/lib/python2.7/dist-packages/pyicloud/services/photos.py", line 457, in asset_date
    tz=pytz.utc)
ValueError: year is out of range
```

**Comment by lapint** [comment]
The issue still exists, however I was able to get past my issue by following another users advice.

@evenwebb:
```
I had this issue. I created a smart album in Photos on Mac to display all
photos older than 1990 and deleted the ones that came up as they had
corrupted dates. The years were set to 0000. It was some filter app that I
had used years back.

Hope this helps :)
```

**Comment by ndbroadbent** [comment]
Fixed in #184
